### PR TITLE
Edit user profile

### PR DESCRIPTION
**Pull request by ducbet**

### CODE REVIEW

File: app/helpers/sessions_helper.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+module SessionsHelper
+
+  def current_user
+    @current_user = User.first
```
Comment (thaotonto) [inline app/helpers/sessions_helper.rb]: hàng fake rồi

File: app/views/users/edit.html.erb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+<h1><%= t ".update" %></h1>
+
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+     <%= render "form" %>
+
+     <div class="gravatar_edit">
+      <%= gravatar_for @user %>
+      <%= link_to t(".change"), "http://gravatar.com/emails",
+                                target:"_blank", rel:"noopener" %>
```
Comment (thaotonto) [inline app/views/users/edit.html.erb]: convention

**Comment by thaotonto** [comment]
ok

**Comment by noobsi** [comment]
ok

**Comment by ducbet** [comment]
image

**Pull request merged**